File: sources/paopao/Boost/AdBoost.cs
using System;

public struct AdBoost
{
	public void Initialize()
	{
		this.validTrigger = false;
		this.validTime = 0f;
		this.useLeftTime = 0f;
	}

	public bool validTrigger;

	public float validTime;

	public float useLeftTime;
}

File: src/shared/kartEngine/Boost/AdBoost.ts
export class AdBoost {
    public validTrigger: boolean = false;
    public validTime: number = 0.0;
    public useLeftTime: number = 0.0;

    constructor() {
        this.Initialize();
    }

    public Initialize(): void {
        this.validTrigger = false;
        this.validTime = 0.0;
        this.useLeftTime = 0.0;
    }
}

export default AdBoost;